palm/plugins/dbt/dbt_containerizer.py

```diff
@@ -146,3 +95,33 @@ def validate_python_version(self) -> bool:
             True
         """
         return True
+
+    @classmethod
+    def determine_profile_strategy(cls, project_path:"Path") -> Tuple[str,str]:
+        """determines where the on-the-host project 
+           has been storing the profiles.yml file
+
+           Args:
+            project_path: the project root to convert
+           Returns:
+              the host and container volume mount values
+        """
+        container_default = "/root/.dbt"
+        if profile_path := os.getenv("DBT_PROFILES_DIR"):
+            profiles_dir = Path(profile_path)
+            if not profiles_dir.exists():
+                raise AbortPalm("Your host has a non-existant DBT_PROFILES_DIR value!")
+            if project_path in profiles_dir.parents:
+                def return_relative(prefix:str) -> str:
+                    return str(profiles_dir).\
```

Comment:
This is way more readable as a single line 

```
return str(profiles_dir).replace(str(project_path), prefix)
```

Reply:
k will update!